docs/project/set-up-dev-env.md

```diff
@@ -22,8 +22,8 @@ You use the `docker` repository and its `Dockerfile` to create a Docker image,
 run a Docker container, and develop code in the container. Docker itself builds,
 tests, and releases new Docker versions using this container.
 
-If you followed the procedures that <a href="/opensource/project/set-up-git/" target="_blank">
-set up Git for contributing</a>, you should have a fork of the `docker/docker`
+If you followed the procedures that [set up Git for contributing](/opensource/project/set-up-git.md),
```

Comment:
I think `/opensource/project/` should be removed here, but be sure to test if the link is generated after updating